From c4ec25791225afd7371d6ebbff41c7c4e49088bc Mon Sep 17 00:00:00 2001
From: Nikolay Aleksandrov <nikolay@redhat.com>
Date: Fri, 5 Sep 2014 11:03:49 -0400
Subject: [ethernet] sfc: Rename 'use_options' variable in tso_start() to clearer 'use_opt_desc'

Message-id: <1409915048-26438-14-git-send-email-nikolay@redhat.com>
Patchwork-id: 91416
O-Subject: [RHEL7.1 PATCH BZ 1110888 13/32] sfc: Rename 'use_options' variable in tso_start() to clearer 'use_opt_desc'
Bugzilla: 1110888
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Ivan Vecera <ivecera@redhat.com>
RH-Acked-by: Stefan Assmann <sassmann@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

From: Ben Hutchings <bhutchings@solarflare.com>

Signed-off-by: Ben Hutchings <bhutchings@solarflare.com>
Signed-off-by: Shradha Shah <sshah@solarflare.com>
Signed-off-by: David S. Miller <davem@davemloft.net>
(cherry picked from commit 93413f5058aa129bd6134b3176633a5ce198567b)
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/net/ethernet/sfc/tx.c b/drivers/net/ethernet/sfc/tx.c
index 126e4ea..ede8dcc 100644
--- a/drivers/net/ethernet/sfc/tx.c
+++ b/drivers/net/ethernet/sfc/tx.c
@@ -1014,7 +1014,7 @@ static void efx_enqueue_unwind(struct efx_tx_queue *tx_queue)
 static int tso_start(struct tso_state *st, struct efx_nic *efx,
        const struct sk_buff *skb)
 {
- bool use_options = efx_nic_rev(efx) >= EFX_REV_HUNT_A0;
+ bool use_opt_desc = efx_nic_rev(efx) >= EFX_REV_HUNT_A0;
  struct device *dma_dev = &efx->pci_dev->dev;
  unsigned int header_len, in_len;
  dma_addr_t dma_addr;
@@ -1040,7 +1040,7 @@ static int tso_start(struct tso_state *st, struct efx_nic *efx,
 
  st->out_len = skb->len - header_len;
 
- if (!use_options) {
+ if (!use_opt_desc) {
   st->header_unmap_len = 0;
 
   if (likely(in_len == 0)) {
-- 
1.7.1